src/abstractdataframe/abstractdataframe.jl

```diff
@@ -390,20 +390,23 @@ describe(io, df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissi
 * `io` : optional output descriptor
 * `stats::AbstractVector{Symbol}`: the summary statistics to report. Allowed 
   fields are `:mean`, `:std`, `:min`, `:q25`, `:median`, `:q75`, `:max`, `:eltype`, 
-  `:nunique`, and `:nmissing`
+  `:nunique`, `first', `last`, and `:nmissing`
```

Comment:
Should be `:first` and `:last`. Also the first closing backtick isn't a real one.